crates/optimism/primitives/src/bedrock.rs

```diff
@@ -43,12 +43,12 @@ pub const BLOCK_NUMS_REPLAYED_TX: [u64; 6] = [
 /// with replayed transaction happen to only contain the single transaction.
 pub fn is_dup_tx(block_number: u64) -> bool {
     if block_number > BLOCK_NUMS_REPLAYED_TX[5] {
-        return false
+        return false;
```

Comment:
pls revert unrelated formatting changes like this one

Reply:
https://github.com/paradigmxyz/reth/pull/12537/commits/ad5150ea788ceb5aff86cbb3994db34a6e652c6f